app/src/main-process/main.ts

```diff
@@ -729,7 +729,13 @@ function createWindow() {
       electron: '>=1.2.1',
     }
 
-    const extensions = [REACT_DEVELOPER_TOOLS, ChromeLens]
+    const axeDevTools = {
+      id: 'lhdoppojpmngadmnindnejefpokejbdd',
+      electron: '>=1.2.1',
```

Comment:
Copied this electron line from the ChromeLens extension above. If I do not have it, the extension fails to load with an error saying the electron version doesn't make the required undefined range. Thus, that version number is arbitrary.